.buildkite/notebook-parameters.yml

```
@@ -1,7 +1,18 @@
 ---
 # configure the parameters replaced in jupyter notebooks when running on CI with papermill
-epochs: 2
-walk_length: 5
-batch_size: 5
+# common parameters used in several notebooks:
+epochs: 1
+batch_size: 2
```

Decreasing the batch size doesn't necessarily make things faster... in fact, it can make things slower, because there's more per-batch overhead. Decreasing sometimes seems to make the accuracy increase faster, but I think that's not something we care about for this testing?

Did this decrease seem to help? If not, maybe we could leave it as it was

```suggestion
batch_size: 5
```

and maybe even investigate increasing it in a follow-up PR?

---

batch size is now back to 5, decrease didn't help.